### PR TITLE
Conditional URLs

### PR DESCRIPTION
**Pull request by sp1tz**
Base on if the app is running in production, use different URLs.

### CODE REVIEW

File: frontend/src/secrets.ts [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
-        return 'https://us-central1-arjunguha-research-group.cloudfunctions.net/ocelot-beta/';
-    }
-}
-
-export const LOGIN_CLIENT_ID: string = '692270598994-p92ku4bbjkvcddouh578eb1a07s8mghc.apps.googleusercontent.com';
-export const MODULE_WL_URL: string = 'https://raw.githubusercontent.com/umass-compsci220/ocelot-settings/master/libraries.json';
+const isProd: boolean = window.location.hostname === 'www.ocelot-ide.org';
+export const CLD_FN_BASE_URL: string = `https://us-central1-arjunguha-research-group.cloudfunctions.net/${isProd ? 'paws' : 'ocelot-beta'}/`,
+             LOGIN_CLIENT_ID: string = '692270598994-p92ku4bbjkvcddouh578eb1a07s8mghc.apps.googleusercontent.com',
+             MODULE_WL_URL: string = `https://raw.githubusercontent.com/umass-compsci220/ocelot-settings/master/libraries${isProd ? '' : '-beta'}.json`;
```
Comment (sp1tz) [inline frontend/src/secrets.ts]: My thinking here is that we will commit directly to `master` in "ocelot-settings" to provide it with a mapping of modules that exist on a branch.

File: frontend/src/secrets.ts [hunk 2]
```diff
@@ -1,11 +1,4 @@
...
-    if (window.location.hostname === 'www.ocelot-ide.org') {
-        return 'https://us-central1-arjunguha-research-group.cloudfunctions.net/paws/';
-    } else {
-        return 'https://us-central1-arjunguha-research-group.cloudfunctions.net/ocelot-beta/';
-    }
-}
-
-export const LOGIN_CLIENT_ID: string = '692270598994-p92ku4bbjkvcddouh578eb1a07s8mghc.apps.googleusercontent.com';
-export const MODULE_WL_URL: string = 'https://raw.githubusercontent.com/umass-compsci220/ocelot-settings/master/libraries.json';
+const isProd: boolean = window.location.hostname === 'www.ocelot-ide.org';
```
Comment (arjunguha) [inline frontend/src/secrets.ts]: Good. This is simpler for clients.

File: frontend/src/secrets.ts (hunk 1 above)
Comment (arjunguha) [reply frontend/src/secrets.ts]: SGTM

**Comment by sp1tz** [comment]
> Actually, this commit is not yet ready. It will be ready when lib220.ts is deleted from this branch (and tests that use lib220 work).

Can't this change can be merged independently? The work you describe is for EJS.

**Pull request merged**